WcaOnRails/public/images/organizations/swisscubing.svg

```diff
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+  "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg width="336" height="334">
+  <image id="image0" width="336" height="334" x="0" y="0"
+    xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAVAAAAFOCAYAAADQNEUyAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
```

Comment:
This looks like it's just an SVG with an embedded PNG. Can we just upload the PNG instead?